custom-connectors/My Acclaro/README.md

```diff
@@ -0,0 +1,81 @@
+# My Acclaro
```

Comment:
@msanz-acclaro You have placed your connector under "custom-connectors" which is not going to be deployed and published into any production region. 

Docs - https://github.com/microsoft/PowerPlatformConnectors#microsoft-power-platform-connectors

Reply:
@sriyen-msft moved from "custom-connectors" to "certified-connectors" as the intention is to get the connector certified. **Please accept my apologies**, the [documentation at MS](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission) is a bit misleading on this topic.

Reply:
moved to certified-connectors